batavia/utils.js

```diff
@@ -791,3 +787,29 @@ batavia.make_callable = function(func) {
     fn.__python__ = true;
     return fn;
 };
+
+/*************************************************************************
+ * get the argument type as a string
+ *************************************************************************/
+ 
+ batavia.get_type = function(arg) {
```

Comment:
Oh - wait - this is to get a printable version of a type, rather than the "type" itself. In that case, there's probably something we can do about actually making the class itself have a `__str__` method... but that's a much bigger job, so feel free to ignore that for now. The only fix I'd suggest is to rename to `get_type_name` so it's clear that it *is* a string, not something that should be a type descriptor.